inference-engine/tests_deprecated/functional/shared_tests/transformations/fully_connected_test.cpp

```diff
@@ -0,0 +1,240 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
I strongly against contributing to deprecated tests.  `tests_deprecated` was considered to be read-only. New test cases have to be added only to the `tests` folder and deprecated tests have to be migrated to new tests infra if new test cases are required.